shanoir-ng-users/src/main/java/org/shanoir/ng/email/EmailServiceImpl.java

```diff
@@ -205,9 +205,8 @@ public void notifyCreateAccountRequest(final User user, final String password) {
 			final String content = build("notifyCreateAccountRequest", variables);
 			messageHelper.setText(content, true);
 		};
-		LOG.error("User asked for an account" + user.getUsername());
+		LOG.info("User asked for an account: " + user.getUsername());
```

Comment:
Better use `{}` here, instead of string concatenation

Reply:
done